drivers/sx127x/sx127x_netdev.c

```diff
@@ -444,8 +444,8 @@ static int _set(netdev_t *netdev, netopt_t opt, const void *val, size_t len)
         return sizeof(netopt_enable_t);
 
     case NETOPT_RX_SYMBOL_TIMEOUT:
-        assert(len <= sizeof(uint16_t));
-        sx127x_set_symbol_timeout(dev, *((const uint16_t *)val));
+        assert(len <= sizeof(uint8_t));
```

Comment:
Huh? `assert(len <= 1)? symbol timeout is max 1 symbol?

Reply:
I think this is just making sure that the value is `uint8_t`, as it is checking `len`, not `val`.

Reply:
> I think this is just making sure that the value is uint8_t, as it is checking len, not val.

Yes. This just checks that the data structure that holds `val` is not greater than 1 byte (uint8t_t)